### ISSUE TITLE
Travis build broken adobe gateway setup

### ISSUE THREAD

**Issue by csantanapr** [apache/incubator-openwhisk-devtools#98]
@ddragosd We can't merge any PRs until Travis is fixed.

master fails like this: https://travis-ci.org/apache/incubator-openwhisk-devtools/builds/344171260
```
mkdir -p ~/tmp/openwhisk/apigateway/ssl
cd ./openwhisk-master/ansible/roles/nginx/files/ && ./genssl.sh 10.20.1.242 server
/bin/sh: 1: ./genssl.sh: not found
make: *** [setup] Error 127
The command "${TOOL}/.travis/build.sh" exited with 2.
Done. Your build exited with 1.
```

look like is configured to use an adobe gateway `image: adobeapiplatform/apigateway:1.1.0` https://github.com/apache/incubator-openwhisk-devtools/blob/master/docker-compose/docker-compose.yml#L130

I think the `docker-compose` should be updated to use the 

I think it should be change to use the regular `openwhisk/apigateway` or make it default to use the openwhisk and then have an option to use the alternate image gateway

**Comment by ddragosd** [comment]
Fixed this in #99

**Comment by ddragosd** [comment]
The build is passing right now so I just need help with a review, then we can unblock the other PR.

**Comment by csantanapr** [comment]
Closed by #99
